sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/processor/CompactBlockProcessor.kt

```diff
@@ -1431,6 +1431,24 @@ class CompactBlockProcessor internal constructor(
                     }.map { scanResult ->
                         Twig.debug { "Scan stage done with result: $scanResult" }
 
+                        var resultState =
+                            if (scanResult.stageResult == SyncingResult.ScanSuccess) {
+                                // TODO: Is this correct?
+                                SyncingResult.AllSuccess
```

Comment:
I think this is correct (in that this PR redefines "all sync stages succeeded" to exclude "deleted scanned blocks from the cache"), but I don't know if this is the only change necessary to implement this redefinition.

Reply:
Yes, that's the correct way here.